### PR TITLE
Gradle + Agones API update

### PR DESCRIPTION
**Pull request by titivermeesch**
Agones updated the requirements for the health check. It now requires an empty object to correctly validate.

I also set up Gradle instead of maven.

### CODE REVIEW

File: src/main/java/dev/agones/AgonesSDK.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public void startHealthTask(int period, TimeUnit unit) {
     }
 
     public void health() {
-        getEndpoints(AgonesEndpoints.class).health();
+        getEndpoints(AgonesEndpoints.class).health(new Object());
```
Comment (MylesDev) [inline src/main/java/dev/agones/AgonesSDK.java:67]: An `Empty` model class would be better here. (See models package)

**Review by MylesDev — COMMENTED** [review]
I'm not sure removing Maven in favour of Gradle is in the scope of this pull request, as ideally it should be to support the API changes. 

I think a seperate PR for Gradle support should be added, personally. Do you have any thoughts, @theminecoder?

**Comment by titivermeesch** [comment]
> I'm not sure removing Maven in favour of Gradle is in the scope of this pull request, as ideally it should be to support the API changes.
> 
> I think a seperate PR for Gradle support should be added, personally. Do you have any thoughts, @theminecoder?

Yes I agree on that one as well. I'll update the PR accordingly

File: src/main/java/dev/agones/AgonesSDK.java (hunk 1 above)
Comment (titivermeesch) [reply src/main/java/dev/agones/AgonesSDK.java:67]: Which `Empty` model are you talking about?

Comment (MylesDev) [reply src/main/java/dev/agones/AgonesSDK.java:67]: You'd need to add it :)

**Comment by titivermeesch** [comment]
Sorry, closing this one. I messed up my local environment pretty badly, so completely deleted the fork. New PR is coming in soon

**Pull request closed without merging**